samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -24,6 +24,14 @@ LOG_MODULE_REGISTER(raw_tx_packet, CONFIG_LOG_DEFAULT_LEVEL);
 #define IEEE80211_SEQ_CTRL_SEQ_NUM_MASK 0xFFF0
 #define IEEE80211_SEQ_NUMBER_INC BIT(4) /* 0-3 is fragment number */
 
+enum rate_flags {
+	LEGACY = 0,
+	FORMAT_11N,
+	FORMAT_VHT,
+	HE_SU,
+	HE_ER_SU
+};
```

Comment:
https://github.com/nrfconnect/sdk-zephyr/blob/main/include/zephyr/net/wifi.h so that multiple samples can use this? (I see that this is needed even in samples/wifi/shell/src/wifi_raw_tx_pkt_shell.c)

Reply:
As we are handling rate value in Kconfig, validation for data rate is not needed in code. So, removing the enum.